Sources/Foundation/Process.swift

```diff
@@ -1165,3 +1158,11 @@ extension Process {
     
     public static let didTerminateNotification = NSNotification.Name(rawValue: "NSTaskDidTerminateNotification")
 }
+
+#if os(Windows)
```

Comment:
Can you check if this is fixed on `main` per discussion?

Reply:
Checked just now. Not yet.